Assets/Scripts/Controllers/AI/Enemy/Nodes/CheckPlayerInRange.cs

```diff
@@ -14,6 +14,7 @@ public class CheckPlayerInRange : Node {
 		if (target == null) {
 			Collider2D[] colliders = Physics2D.OverlapCircleAll(tree.transform.position, tree.Stats.DetectionRange, _enemyLayerMask);
 			if (colliders.Length > 0) {
+				//tree.target is unnecessary because at the beginning you say Target = target
```

Comment:
Sounds more like something for a PR discussion rather than a code comment :P

Reply:
Toine's comment, which is also incorrect as tree.target is necessary to actually assign the target to the tree instead of a local variable.